include/Core/Posts/PostsMetabox.php

```diff
@@ -251,4 +251,67 @@ protected function delete_posts_by_id( $post_ids, $force_delete ) {
 
 		return count( $post_ids );
 	}
+
+	/**
+	 * Generate display name from post type and status.
+	 *
+	 * @static
+	 *
+	 * @param string $str
+	 *
+	 * @return string Label
+	 */
+	public static function display_post_type_status( $str ) {
+		$type_status = self::split_post_type_status( $str );
+
+		$status = $type_status['status'];
+		$type   = $type_status['type'];
+		$label  = '';
+
+		switch ( $status ) {
+			case 'private':
+				$label = $type . ' - Private Posts';
+				break;
+			case 'future':
+				$label = $type . ' - Scheduled Posts';
+				break;
+			case 'draft':
+				$label = $type . ' - Draft Posts';
+				break;
+			case 'pending':
+				$label = $type . ' - Pending Posts';
+				break;
+			case 'publish':
+				$label = $type . ' - Published Posts';
+				break;
+		}
+
+		return $label;
+	}
+
+	/**
+	 * Split post type and status.
+	 *
+	 * @static
+	 * @access public
+	 *
+	 * @param string $str
+	 *
+	 * @return array
+	 */
+	public static function split_post_type_status( $str ) {
```

Comment:
Make this function as protected and non-static.

You can also remove this function from BDUtil class